tests/Doctrine/Tests/Common/Proxy/ProxyLogicTest.php

```diff
@@ -193,13 +192,16 @@ static function () use ($test) {
         self::assertSame('publicAssociationValue', $this->lazyObject->publicAssociation);
     }
 
+    /**
+     * @requires PHP > 7.1
```

Comment:
`expectNotice` does not exist in PHPUnit running on 7.1. I figured the test can be skipped as this is tested in higher versions anyway but I can also check for `expectNotice` existence and conditionally call old method.